### DISCUSSION TITLE
Nav tab on refresh

### DISCUSSION THREAD

**Discussion by mariusares** [twbs/bootstrap#36433] [Q&A]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

Hi guys
I have a nav tab menu with one tab active by default.
I'm trying to set the website to display at the same tab what was opened, on refresh.
It is changing the tabs on click but it goes to the default tab on refresh.
Here is what i have tryed:
<script>
$(document).ready(function(){
    $('a[data-bs-toggle="tab"]').on('show.bs.tab', function(e) {
        localStorage.setItem('activeTab', $(e.target).attr('href'))
    });
    var activeTab = localStorage.getItem('activeTab');
    if(activeTab){
        $('#nav-tab a[href="' + activeTab + '"]').tab('show');
    }
});
</script>

### Reduced test cases

https://www.tutorialrepublic.com/faq/how-to-keep-the-current-tab-active-on-page-reload-in-bootstrap.php

### What operating system(s) are you seeing the problem on?

Linux

### What browser(s) are you seeing the problem on?

Firefox

### What version of Bootstrap are you using?

5.2

**Comment by mdo** [comment]
The proper place for this kind of question is in our Discussions, not an issue (as this isn't a bug report or feature request).
